torch/nn/modules/transformer.py

```diff
@@ -171,11 +173,10 @@ def forward(self, src, mask=None, src_key_padding_mask=None):
         """
         output = src
 
-        for i in range(self.num_layers):
```

Comment:
So `range` is not jitable as well?

Reply:
`ModuleList`s can't be indexed, but can be used with `for-in` loops. This should be the same